content/en/docs/concepts/services-networking/dual-stack.md

```diff
@@ -15,7 +15,7 @@ weight: 70
 
 {{% capture overview %}}
 
-{{< feature-state for_k8s_version="v1.16" state="alpha" >}}
+{{< feature-state for_k8s_version="v1.18" state="alpha" >}}
```

Comment:
This change has the meaning that the original documents were incorrect and dual-stack did not go alpha until v1.18

Is that correct?

Reply:
That is incorrect and wasn't my intent. This enhancement went into alpha in 1.16 and is still alpha in 1.18. I was trying to make that clear in the documentation as I have been asked a number of times if this is beta. If this isn't the change that needs to happen then I will remove it. Is there any way to say that it's still in alpha in 1.18?

Reply:
We could change either the hover information or initial rendering for the shortcode. @lachie83 it sounds like you've seen a shortcoming on how the current rendering presents the feature state information. The website build process knows that, for example, the site being built is v1.18 docs and the state change to alpha happened in v1.16 - however, that's not all presented in the page text itself.

Would you be willing to rephrase that in your own words and file it as an issue?